scene/main/node.h

```diff
@@ -414,6 +418,7 @@ class Node : public Object {
 
 #ifndef DISABLE_DEPRECATED
 	void _set_name_bind_compat_76560(const String &p_name);
+	const Variant _get_rpc_config_bind_compat_106848();
```

Comment:
You still need to add this line to `misc/extension_api_validation/4.4-stable.expected`:
```
Validate extension JSON: API was removed: classes/Node/methods/get_rpc_config
```

This serves as documentation for the compat breakage, even if it's all fine thanks to compat bindings.